exams/Sum.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+using namespace std;
+int main() {
+    int i;
+    float Sum = 0, A[i];
+    cout<<"Enter size massive = "<<endl;
+    cin>>i;
+    cout<<"Massive: "<<endl;
+    for (int j = 0; j < i; j++){
+        cin>>A[i];
+        Sum = Sum + A[i];

```

Comment:
```suggestion
        Sum += A[i];
```
It'll be better to replace asignment "=" with "+="